src/components/regions/Regions.js

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from 'react';
+import { shallowEqual, useSelector, useDispatch } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import Pagination from 'react-bootstrap/Pagination';
+import Region from './Region';
+import Header from '../Header';
+import Loading from '../Loading';
+import { fetchCountriesAction } from '../../redux/countries/countries';
+
+const Regiones = () => {
+  const { id } = useParams();
+  const dispatch = useDispatch();
+  const { countries, loaded } = useSelector((state) => state.countries, shallowEqual);
+  const [regions, setRegions] = useState(null);
+  const [activePage, setActivePage] = useState(1);
+
+  useEffect(() => {
+    dispatch(fetchCountriesAction());
+    if (loaded) {
+      const countryObj = countries.filter((obj) => obj.id === id);
+      // Non camelCase are needed here since they're used by the Narrativa API.
+      /* eslint-disable camelcase */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean as recommended.

Reply:
If I remove it it won't response to the API since No camelCase are used.

Reply:
It's for the API call, No camelCase are used

Reply:
look at these suggestion I've left https://github.com/oyelakinG9/unique-data/pull/1#discussion_r825329360

Reply:
look at this suggestion I left https://github.com/oyelakinG9/unique-data/pull/1#discussion_r825329360